templates/publisher/account-snaps.html

```diff
@@ -0,0 +1,166 @@
+{% extends "_layout.html" %}
+
+{% block title %}
+My snaps — Linux software in the Snap Store
+{% endblock %}
+
+{% block content %}
+<div class="p-strip is-shallow u-no-padding--bottom">
+  <div class="row">
+    <div class="col-12">
+      <h1 class="p-heading--three u-float--left">My snaps</h1>
+      <a href="https://docs.snapcraft.io/build-snaps/" class="p-link--external u-float--right">How to create a snap</a>
+    </div>
+  </div>
+  <div class="row u-no-margin--top">
+    <div class="col-12">
+      <hr {% if not snaps %}class="u-no-margin--bottom" {% endif %}/>
```

Comment:
Is this `hr` meant to be here at all when there is no snaps? It looks weird with light strip just below it:

<img width="1379" alt="screen shot 2018-04-13 at 11 26 01" src="https://user-images.githubusercontent.com/83575/38727906-9b2bd590-3f0e-11e8-8248-19130670a4c5.png">